Nickvision.Aura.Tests/UpdaterTest.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Nickvision.Aura.Update;
+using System;
+using System.Threading.Tasks;
+
+namespace Nickvision.Aura.Tests;
+
+public class UpdaterTest
+{
+    [Fact]
+    public async Task LatestUpdateTest()
+    {
+        var currentVersion = new Version("2023.8.0");
+        Aura.Init("org.nickvision.test", "Nickvision Test");
+        Aura.Active.AppInfo.SourceRepo = new Uri("https://github.com/NickvisionApps/Denaro");
+        var updater = new Updater();
+        var latestUpdate = await updater.GetCurrentStableVersionAsync();
+        Assert.NotNull(latestUpdate);
+        Assert.True(latestUpdate > currentVersion);
+    }
+}
```

Comment:
I would make it a `SkippableFact` and skip the test somehow if there's no internet connection

Reply:
I skipped the test if the version from `GetCurrentStableVersionAsync` is null which would happen if there is no internet. The only other time it would happen is if `https://github.com/NickvisionApps/Denaro` doesn't exist and we know it does haha.